proca/lib/proca/stage/message_v2.ex

```diff
@@ -59,6 +46,38 @@ defmodule Proca.Stage.MessageV2 do
     |> put_action_meta(stage)
   end
 
+  def action_page_data(
+        %ActionPage{
+          locale: locale,
+          name: name,
+          thank_you_template: thank_you_template,
+          supporter_confirm_template: supporter_confirm_template,
+          org: org
+        } = action_page
+      ) do
+    %{
+      "locale" => locale,
+      "name" => name,
+      "thankYouTemplate" => thank_you_template,
```

Comment:
We don't need anything related to templates (thankYouTemplate, thankYouTemplate, supporterConfirmTemplate)